### ISSUE TITLE
💬 RFC: Login/Logout hidden commands

### ISSUE THREAD

**Issue by JacobMGEvans** [cloudflare/wrangler2#697]
Login and Logout are top level commands that can have high usage. Currently the Login/Logout commands are hidden commands in the help message but will be showed in the documentation website. This seems like a incontinuity and may cause confusion if user is looking for the commands in the CLI help message.

**Comment by threepointone** [comment]
Yeah good idea. Let's also group the user specific commands (login/logout/whoami) under a `User` header.

**Comment by JacobMGEvans** [comment]
> Yeah good idea. Let's also group the user specific commands (login/logout/whoami) under a `User` header.

It seems like Yargs `group()` only works for flags currently. They have an issue that has been open for 3 years for a command group function. https://github.com/yargs/yargs/issues/684
